Update YOLACT integration docs (#976)
neuralmagic/sparseml: integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md

1. For this tutorial, run the COCO setup script with the following command from the root of the `yolact` repository:
    ```bash
    sparseml.yolact.download
    ```
   

>     Download and validation of the COCO dataset will begin and take around 10 minutes to finish (based on the speed of your internet connection).

Reviewer: ```suggestion
Based on your internet connection, downloading and validation of the COCO dataset will take around 10 minutes to complete.
```

    The script downloads the COCO dataset into a `coco` folder under the data directory.

>     Notice that once completed, the data is ready for training with the folder structure in the following state (only directories are shown for brevity):

Reviewer: ```suggestion
Once completed, the data is ready for training with the folder structure in the following state (only directories are shown for brevity):
```

    ```
    ├── data

>        ├── coco

Reviewer 2: nit: Should this be inward indented?

1. Training YOLACT from scratch requires a pretrained-backbone model, 
   currently SparseML supports training with `DarkNet-53` backbone. 
   - Download ImageNet-pre-trained [`Darknet-53` backbone](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing)
     and put it in `./weights` directory. The directory structure should look like the following:
````
    └── Project_directory
        ├── data
        │   ├── coco
        │   │   ├── annotations
        │   │   └── images
        │   └── scripts
        └── weights
            └── darknet53.pth
````

2) Run the following command to kickstart training
```bash
sparseml.yolact.train --resume \
./weights/darknet53.pth \
--train_info ./data/coco/annotations/instances_train2017.json \
--validation_info ./data/coco/annotations/instances_val2017.json \
--train_images ./data/coco/images \

> --validation_images ./datas/coco/images

Author: ```suggestion
--validation_images ./data/coco/images
```

>    - Download ImageNet-pre-trained [`DarkNet-53` backbone](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing)

Reviewer: ```suggestion
   - Download ImageNet-pre-trained [`Darknet-53` backbone](https://drive.google.com/file/d/17Y431j4sagFpSReuPNoFcj9h7azDTZFf/view?usp=sharing)
```

3. To begin applying one of the recipes, use the `--recipe` argument within the YOLACT [train script](https://github.com/neuralmagic/yolact/blob/master/train.py).
   The recipe argument is combined with our previous training command and COCO pre-trained weights to run the recipes over the model. For example, a command for pruning YOLACT would look like this:
```bash
sparseml.yolact.train \
--recipe=../recipes/yolact.pruned.md \
--resume=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none \
--save_folder=./pruned \
--train_info ./data/coco/annotations/instances_train2017.json \
--validation_info ./data/coco/annotations/instances_val2017.json \
--train_images ./data/coco/images \

> --validation_images ./datas/coco/images

Author: ```suggestion
--validation_images ./data/coco/images
```
